pull_request_template.md

```diff
@@ -48,3 +48,6 @@ This change added tests and can be verified as follows:
   - Does this pull request introduce a new feature or user-facing behavior changes? (yes / no)
   - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (yes / no)
   - How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  / README.md  /   not documented)
+  - Does this pull request update existing proto field values and require a migration? (yes / no)
```

Comment:
nit: but if the field names change, that could also require a migration. Maybe just generalize it to "Does this pull request update existing proto data types and require a migration?"

Reply:
why would a field name change require a migration?

Reply:
ah yeah I guess if just the name changed, it wouldn't matter. But if you added a field, it could require a migration (if you have to backfill)

Reply:
I think how you have it is fine! I was interpreting value as the stored value in state - not the "proto value" (i.e. the index)